plutus-contract-certification/plutus-contract-certification.cabal

```diff
@@ -0,0 +1,47 @@
+cabal-version: 3.0
+name: plutus-contract-certification
+version: 0.1.0.0
+license: Apache-2.0
+license-files:
+  LICENSE
+  NOTICE
+maintainer: jann.mueller@iohk.io
```

Comment:
Not fair for Jann to have to maintain this :-). Max or Ulf, who volunteers with their name here?